src/Illuminate/Support/Collection.php

```diff
@@ -280,7 +280,7 @@ public function dd()
     /**
      * Dump the collection.
      *
-     * @return void
+     * @return $this
```

Comment:
Sorry about that. I haven't seen the $this keyword in a while for a return type and it looked super off. Still seems weird that they added one in with the $ while all the others don't have any special characters  😬

https://phpdoc.org/docs/latest/guides/types.html#keywords